moveit_kinematics/cached_ik_kinematics_plugin/README.md

```diff
@@ -0,0 +1,80 @@
+# Cached IK Kinematics Plugin
```

Comment:
Great documentation!

Reply:
Once it's merged, perhaps a summary of this (or simply a pointer) can be added to this page: http://docs.ros.org/kinetic/api/moveit_tutorials/html/doc/pr2_tutorials/kinematics/src/doc/kinematics_configuration.html

Reply:
Yes, I'd speculate this won't get used much if it isn't added to the tutorials